src/index.js

```diff
@@ -1 +1,41 @@
-console.log('Hello');
\ No newline at end of file
+const fs = require('fs');
+const path = require('path');
+const csv = require('csvtojson');
+
+const main = async () => {
```

Comment:
Consider breaking this function down into smaller logically separated functions to improve overall readability of the code.

Reply:
I've broke down the function into smaller functions and placed them in a utils folder.
I've also renamed (for better I think) the main function.